typescript/nomad-deploy/src/core/index.ts

```diff
@@ -676,6 +676,132 @@ export async function deployNChains(deploys: CoreDeploy[]) {
   }
 }
 
+
+/**
+ * Deploy the entire suite of Nomad contracts
+ * on each chain within the chainConfigs array
+ * including the upgradable Home, Replicas, and GovernanceRouter
+ * that have been deployed, initialized, and configured
+ * according to the deployNomad script
+ *
+ * @dev The first chain in the array will be the governing chain
+ *
+ * @param deploys - An array of chain deploys
+ */
+export async function deployHubAndSpoke(hub: CoreDeploy, spokes: CoreDeploy[]) {
```

Comment:
because you reverse the order of `gov, non` in other functions, this one is now inconsistent by putting the hub first

Reply:
https://github.com/nomad-xyz/nomad-monorepo/pull/3/commits/30a0b0d23e7171f6c01fc89835659545bca08415